app/models/user.rb

```diff
@@ -429,6 +429,28 @@ def depressed_reports
       ).pluck(:last_sad_report_id)
       Report.joins(:user).where(id: ids).order(reported_on: :desc)
     end
+
+    # FIXME: 一次対応として一回でも休会している受講生にはメッセージ送信済みとする
+    #        別Issueで入会n日目、休会開けn日目目の受講生にメッセージを送信する方針へ改修してほしい
+    #        改修後、このメソッドは不要になると思われるので削除すること
+    def mark_message_as_sent_for_hibernated_student
+      User.find_each do |user|
+        if user.hibernated?
+          user.assign_attributes(sent_student_followup_message: true)
+          user.save(validate: false)
+        end
+      end
+    end
+
+    def create_followup_comment(from_user, student)
+      from_user.comments.create(
+        description: I18n.t('send_message.description'),
+        commentable_id: Talk.find_by(user_id: student.id).id,
+        commentable_type: 'Talk'
+      )
+      student.assign_attributes(sent_student_followup_message: true)
```

Comment:
```suggestion
      student.sent_student_followup_message = true
```

変更する項目が１つでsaveメソッドで保存するのであればこちらの方がわかりやすいかもです〜

Reply:
@komagata 
なるほど 😲　確かにこちらのほうが読みやすいですね。
ad16de583766cf325a6ad4aef4dfb7d83e3b35d7
こちらで修正させていただきました〜